pkg/server/server.go

```diff
@@ -79,7 +101,13 @@ func outputBufferToWriter(w http.ResponseWriter, metroFile file.File, format str
 }
 
 func getFormat(r *http.Request) (string, error) {
-	format := r.FormValue("format")
+	var format string
+	if r.Header.Get("Content-Type") == "application/json" {
+		format = r.URL.Query().Get("format")
+	} else {
+		format = r.FormValue("format")
+	}
```

Comment:
Awesome! One last thing, but can we update the OpenAPI spec as well? It should just be a parameter. 

https://github.com/moov-io/metro2/blob/master/api/openapi.yaml